app/build.gradle

```diff
@@ -191,9 +197,41 @@ dependencies {
   implementation project(":domain")
   implementation project(":utility")
 }
+repositories {
+  mavenCentral()
```

Comment:
This shouldn't be necessary since we define this in the root Gradle file. Can you try removing it to see if JaCoCo works?

Reply:
Done